namespaces/api-namespace.md

```diff
@@ -125,6 +140,15 @@ Given an API description with following layout:
 }
 ```
 
+## String Pair (object)
```

Comment:
Do not be too much focused on this type – it is merely a helper to keep things dry. Note it is not even an element. As for HTTP headers – I want them to use MSON syntax / refract eventually.